insights-notification-schemas-java/src/main/java/com/redhat/cloud/notifications/jackson/serializer/LocalDateTimeSerializer.java

```diff
@@ -0,0 +1,18 @@
+package com.redhat.cloud.notifications.jackson.serializer;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class LocalDateTimeSerializer extends JsonSerializer<LocalDateTime> {
+
+    @Override
+    public void serialize(LocalDateTime dateTime, JsonGenerator jsonGenerator, SerializerProvider serializerProvider) throws IOException {
+        jsonGenerator.writeString(dateTime.format(DateTimeFormatter.ISO_LOCAL_DATE_TIME));
```

Comment:
Is it OK to serialize with a different formatter than the one used for deserialization?

Reply:
I'm not sure - I guess it's OK at least for an interim period. We could ask the apps in prod (I guess there is only one) to move to a ISO_LOCAL_DATE_TIME representation.